src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -52,7 +52,7 @@ internal static bool IsSolutionFolder(EnvDTE.Project envDTEProject)
             return envDTEProject.Kind != null && envDTEProject.Kind.Equals(VsProjectTypes.VsProjectItemKindSolutionFolder, StringComparison.OrdinalIgnoreCase);
         }
 
-        internal static async Task<bool> ContainsFile(EnvDTE.Project envDTEProject, string path)
+        public static async Task<bool> ContainsFile(EnvDTE.Project envDTEProject, string path)
```

Comment:
Are you making it public for tests? 
You can add internalsvisibleto. 
We've been using that pattern to minimize the API surface area. 
//cc @PatoBeltran

Reply:
This is not just for tests, instead need to use it across other product assembly. 

And I'm not entirely sure of internalvisible pattern to minimize the public surface area for product code, It may sound good just to test projects.

Reply:
Yeah, I was suggesting to use InternalsVisibleTo if it was only used in tests. 

But it's fine if it's not. I just couldn't find the invocation in the changeset.